src/BLE_Server.cpp

```diff
@@ -39,95 +40,72 @@ BLECharacteristic *fitnessMachineIndoorBikeData;
-      NIMBLE_PROPERTY::READ |
-          NIMBLE_PROPERTY::NOTIFY);
+  // HEART RATE MONITOR SERVICE SETUP
+  BLEService *pHeartService          = pServer->createService(HEARTSERVICE_UUID);
+  heartRateMeasurementCharacteristic = pHeartService->createCharacteristic(HEARTCHARACTERISTIC_UUID, NIMBLE_PROPERTY::READ | NIMBLE_PROPERTY::NOTIFY);
 
-  //Power Meter MONITOR SERVICE SETUP
+  // Power Meter MONITOR SERVICE SETUP
   BLEService *pPowerMonitor = pServer->createService(CYCLINGPOWERSERVICE_UUID);
 
-  cyclingPowerMeasurementCharacteristic = pPowerMonitor->createCharacteristic(
-      CYCLINGPOWERMEASUREMENT_UUID,
-      NIMBLE_PROPERTY::READ |
-          NIMBLE_PROPERTY::NOTIFY);
+  cyclingPowerMeasurementCharacteristic = pPowerMonitor->createCharacteristic(CYCLINGPOWERMEASUREMENT_UUID, NIMBLE_PROPERTY::READ | NIMBLE_PROPERTY::NOTIFY);
 
-  BLECharacteristic *cyclingPowerFeatureCharacteristic = pPowerMonitor->createCharacteristic(
-      CYCLINGPOWERFEATURE_UUID,
-      NIMBLE_PROPERTY::READ);
+  BLECharacteristic *cyclingPowerFeatureCharacteristic = pPowerMonitor->createCharacteristic(CYCLINGPOWERFEATURE_UUID, NIMBLE_PROPERTY::READ);
 
-  BLECharacteristic *sensorLocationCharacteristic = pPowerMonitor->createCharacteristic(
-      SENSORLOCATION_UUID,
-      NIMBLE_PROPERTY::READ);
+  BLECharacteristic *sensorLocationCharacteristic = pPowerMonitor->createCharacteristic(SENSORLOCATION_UUID, NIMBLE_PROPERTY::READ);
 
-  //Fitness Machine service setup
+  // Fitness Machine service setup
   BLEService *pFitnessMachineService = pServer->createService(FITNESSMACHINESERVICE_UUID);
 
-  fitnessMachineFeature = pFitnessMachineService->createCharacteristic(
-      FITNESSMACHINEFEATURE_UUID,
-      NIMBLE_PROPERTY::READ |
-          NIMBLE_PROPERTY::WRITE |
-          NIMBLE_PROPERTY::NOTIFY |
-          NIMBLE_PROPERTY::INDICATE);
-
-  BLECharacteristic *fitnessMachineControlPoint = pFitnessMachineService->createCharacteristic(
-      FITNESSMACHINECONTROLPOINT_UUID,
-      NIMBLE_PROPERTY::WRITE |
-          NIMBLE_PROPERTY::NOTIFY |
-          NIMBLE_PROPERTY::INDICATE);
-
-  BLECharacteristic *fitnessMachineStatus = pFitnessMachineService->createCharacteristic(
-      FITNESSMACHINESTATUS_UUID,
-      NIMBLE_PROPERTY::READ |
-          NIMBLE_PROPERTY::WRITE |
-          NIMBLE_PROPERTY::NOTIFY);
-
-  fitnessMachineIndoorBikeData = pFitnessMachineService->createCharacteristic(
-      FITNESSMACHINEINDOORBIKEDATA_UUID,
-      NIMBLE_PROPERTY::READ |
-          NIMBLE_PROPERTY::NOTIFY);
-
-  BLECharacteristic *fitnessMachineResistanceLevelRange = pFitnessMachineService->createCharacteristic(
-      FITNESSMACHINERESISTANCELEVELRANGE_UUID,
-      NIMBLE_PROPERTY::READ);
-
-  BLECharacteristic *fitnessMachinePowerRange = pFitnessMachineService->createCharacteristic(
-      FITNESSMACHINEPOWERRANGE_UUID,
-      NIMBLE_PROPERTY::READ);
+  fitnessMachineFeature = pFitnessMachineService->createCharacteristic(FITNESSMACHINEFEATURE_UUID,
```

Comment:
This might be an area where it may be nice to have specific formatting for this section as the one liners might get a bit too long unless you have an ultrawide display :)

Reply:
Lines are wrapped at 180 chars